tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "sourceMap": true,
     "strictNullChecks": true,
     "strictPropertyInitialization": true,
-    "verbose": true
+    // TODO: Temporarily skipping the type checking of all declaration files in the node_modules directory. More investigations required to resolve the compiling error when setting this option as false
+    "skipLibCheck": true
```

Comment:
This gives me pause--I'd like to better understand what we're skipping by doing this. Are downstream users of our library impacted by this?

Reply:
When `skipLibCheck` is set to `true`, it enables `TypeScript` to bypass type checking for declaration files. We've encountered a scenario where setting this option to `false` results in type inconsistency errors originating from the `intern` dependency. We have updated this dependency to its latest version and found no related issues in its repository. Since this inconsistency stems from the third-party dependency's type declarations, in my opinion, this configuration should not affect our downstream customers.

Reply:
Created this issue [#63 ](https://github.com/amazon-ion/ion-hash-js/issues/63)as a reference point to help identify the root cause in case any typechecking-related issues arise in the future.